openreview/conference/templates/tabsConferenceDecisionsWebfield.js

```diff
@@ -225,6 +225,13 @@ function renderContent(notes, decisionNotes, withdrawnNotes, deskRejectedNotes,
     $('.tabs-container a[href="#desk-rejected-submissions"]').parent().hide();
   }
 
+  //Hide empty decision tabs
+  for (decision in papersByDecision) {
```

Comment:
@zbialecki please review :)

Reply:
I would prefer not to do this. If a decision tab should be hidden then it can be removed from the DECISION_HEADING_MAP and it won't be rendered at all. Otherwise, I would think that all decision options should be visible otherwise people won't know they exist.

Reply:
We need to get the notes first in order to know if the result is empty or not :(

Reply:
Ok, I'm going to remove the reject key from the DECISION_HEADING_MAP

Reply:
I'm not sure what you mean... at this point in the code all the notes have been fetched and categorized by decision.

Reply:
We first render the tabs and then the content.

Reply:
Ah OK yes, but if the decision isn't in the HEADING_MAP then the tab won't be rendered either

Reply:
BTW I'm going to revert this commit.